### PR TITLE
:book: Add AMI for k8s v1.18.1

### PR DESCRIPTION
**Pull request by detiber**
**What this PR does / why we need it**:

- Adds AMI information for the k8s v1.18.1 release

### CODE REVIEW

File: docs/amis.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
   - [Amazon Linux 2](#amazon-linux-2-2)
   - [CentOS 7](#centos-7-2)
   - [Ubuntu 18.04 (Bionic)](#ubuntu-1804-bionic-2)
-- [Kubernetes Version v1.18.0](#kubernetes-version-v1180)
+- [Kubernetes Version v1.18.1](#kubernetes-version-v1180)
```
Comment (ncdc) [inline docs/amis.md:15]:
```suggestion
- [Kubernetes Version v1.18.1](#kubernetes-version-v1181)
```

File: docs/amis.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
   - [Amazon Linux 2](#amazon-linux-2-3)
```
Comment (ncdc) [inline docs/amis.md:16]:
```suggestion
  - [Amazon Linux 2](#amazon-linux-2-2)
```

**Comment by detiber** [comment]
@ncdc thanks for catching that, fixed.

**Pull request merged**